src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/edit/details/main.js

```diff
@@ -407,9 +393,10 @@ define([
                 this.sandbox.emit('sulu.tab.saving');
                 AccountManager.save(data).then(function(savedData) {
                     this.data = savedData;
-                    this.initContactData();
-                    this.setFormData(this.data);
+                    var formData = this.initContactData();
+                    this.setFormData(formData);
                     this.sandbox.emit('sulu.tab.saved', savedData);
+                    this.sandbox.emit('sulu.tab.data.changed', savedData);
```

Comment:
@nnatter lets get rid of sulu.tab.data.changed event. It's easier to just use a flag on the sulu.tab.saved event